### ISSUE TITLE
AssertionError when setting config.dataset.val.with_damaged_actions=false

### ISSUE THREAD

**Issue by axhiao** [karfly/learnable-triangulation-pytorch#125]
Hi, in the paper you mentioned the validation set has been filtered by removing the scenes 'Greeting-2', 'SittingDown-2', 'Waiting-1'. So I just tried removing theses scenes with config item with_damaged_actions=false. But I got the error:  

> [train=False, test=True] ./data/human36m/extra/human36m-multiview-labels-GTbboxes.npy has 2107 samples, but './data/pretrained/human36m/human36m_alg_10-04-2019/checkpoints/0060/results/val.pkl' has 2181.

I digged into the codes and found that you got **_keypoints_3d_pred_** without filtering. Could you help confirm the issue? Thank you!

**Comment by shrubb** [comment]
Hi, unfortunately we lost access to Human3.6M so can't test this. I already don't remember how this val.pkl was generated... But I think you can re-generate it yourself with or without damaged actions by just changing the algebraic model's yaml config a little and then re-evaluating it. After that, if things still don't work, you can tweak around view-dataset.py (e.g. load and visualize val.pkl there) to see what breaks.

**Comment by axhiao** [comment]
@shrubb thank you!
